README.md

```diff
@@ -42,15 +42,16 @@ Binrc supports settings binary version numbers as environment variables. It will
 in the environment to configure the right version.
 
 ```
-HUGO_VERSION=0.19 binrc install spf13/hugo
+HUGO_VERSION=0.94.2 binrc install gohugoio/hugo
 ```
 
 ### Aliases
 
-Binrc keeps a list of aliases to make installing binaries more easy. If a project name is not in `OWNER/NAME` for, Binrc will
+Binrc keeps [a list of aliases](cache/cache.go) to make installing binaries more easy. If a project name is not in `OWNER/NAME` format, Binrc will
 check the list of aliases to try to resolve the project.
 
 This is the current known list:
 
-- hugo: spf13/hugo
+- hugo: gohugoio/hugo
 - gutenberg: keats/gutenberg
+- zola: getzola/zola
```

Comment:
Well… actually Zola is the new repo for Gutenberg (and `keats/gutenberg` redirects to `getzola/zola` …) — so the question is whether the older aliases shoud be even kept?

Reply:
(I actually ended up here trying to sync the versions/aliases with a proposed update to build_images installed software list to streamline the `GUTENBERG_`/`ZOLA_` info to probably make more prominent that <=0.4 was gutenberg and >=0.5 is zola to make it more obvious…)

Reply:
@mbrukman Actually it's well described in:
https://github.com/netlify/binrc/blob/c3a403de965cdf23a9e7fdfa872fa1947777506c/templates/templates.toml#L10-L16

incl. the correct templates so things are probably still working even now, five years later;)

Reply:
So maybe just a hint along the way:

```suggestion
- zola: getzola/zola (formerly gutenberg: keats/gutenberg)
```

would suffice?

Otherwise the rest of the PR is all golden, it's about time to update the outdated refs, good job!